src/ripple/app/ledger/InboundLedger.h

```diff
@@ -99,6 +123,24 @@ class InboundLedger
     LedgerInfo
     deserializeHeader(Slice data, bool hasPrefix);
```

Comment:
Generic comment, just because I came across this code:

1. This should not be a `static` member of this class; it has very little do with the class and it's just here because it's just where it happened to be put at some point. It should be a free function.
2. The `bool` argument to "skip" bytes is ugly. Callers that need to skip the header should just skip the first 4 bytes and pass the correct `Slice` to this function.

No need to act on this, really. it may be nice to document it or, potentially, open an issue so that it can tackled in the future.

Reply:
I've done both (push incoming). Minor thought: it might be nice to prefer static functions in classes as an extra namespace layer, to avoid polluting the `ripple` namespace. I don't feel too strongly right now, but I like when I can tell where a function comes from by reading its call site or the import statements at the top of the file. C++ makes this difficult with header inclusion and huge, open namespaces.